lib/infrastructure/class-wp-rest-request.php

```diff
@@ -446,7 +446,8 @@ public function get_params() {
 		foreach ( $order as $type ) {
 			$params = array_merge( $params, (array) $this->params[ $type ] );
 		}
-
+		
```

Comment:
This line has trailing whitespace, which is causing the standards check to fail :)